Dockerfile

```diff
@@ -37,21 +38,31 @@ RUN pip3 install \
     pytest
 
 #  Hack antlr3 as required by opengeode
-RUN mkdir -p tmp ;	cd /tmp ; wget -q -O - https://download.tuxfamily.org/taste/antlr3_python3_runtime_3.4.tar.bz2 | tar jxpvf - ; cd antlr3_python3_runtime_3.4 ; python3 -m pip install --upgrade .
+RUN mkdir -p tmp \
+    && cd /tmp \
+    && wget -q -O - https://download.tuxfamily.org/taste/antlr3_python3_runtime_3.4.tar.bz2 | tar jxpvf - \
+    && cd antlr3_python3_runtime_3.4 \
+    && python3 -m pip install --upgrade . \
+    && rm -rf /tmp/antlr3_python3_runtime_3.4
```

Comment:
cd .. && rm

to restore workdir

Reply:
No need to restore the workdir. But I removed tmp and added rm after cd ..